.github/dependabot.yml

```
@@ -20,3 +20,13 @@ updates:
       prefix: build(deps)
     labels:
       - dependencies
+
+  - package-ecosystem: gradle
+    directory: /
+    schedule:
+      interval: weekly
+      day: tuesday
+    commit-message:
+      prefix: java(deps)
```

for a conventional commit:

```suggestion
      prefix: deps(java)
```